### ISSUE TITLE
Android deployment?

### ISSUE THREAD

**Issue by propername** [beeware/beeware#38]
Your tutorial is excellent but when I got to the end I realized I couldn't build my application for android.

What will it take to make BeeWare compatible for android too?

**Comment by freakboy3742** [comment]
Good news - we've [got someone working on it right now](https://twitter.com/PyBeeWare/status/1233645620277047297). 

From the end-developer's perspective, you won't need to do anything different - it should be as straightforward as `briefcase create android` then `briefcase run android`.

On the current plan, we believe the Android backend for *Briefcase* should be ready in a week or so; the Android backend for Toga should be ready in about a month. Updating the BeeWare tutorial is part of that work; we'll publish an update to the tutorial as soon as it's ready.

**Comment by Chocrates** [comment]
Great news @freakboy3742,  are there any outstanding issues that the team needs help with?

**Comment by freakboy3742** [comment]
@Chocrates We're a couple of weeks away from being able to make a meaningful call for assistance. The schedule I gave in my previous comment on this ticket didn't account for delays due to COVID-19 - but it's still being worked on. Briefcase support for Android has landed in the trunk (see beeware/briefcase#321); what we're missing now is *Toga* support for Android - so it's difficult to give end-users an app that does something they can test.

Once Toga support lands (our contractor is working on it right now), we'll need people to run the BeeWare tutorial and tell us all the wonderfully creative ways that it breaks :-) Then we'll need people to help develop widgets and other pieces of Android support.

tldr; we're almost there; watch [Twitter](https://twitter.com/pybeeware) and the [BeeWare blog](https://beeware.org/news) for announcements. I assure you we won't be quiet when it starts working :-)

**Comment by Chocrates** [comment]
Awesome @freakboy3742, thanks for all that you and the team do!

**Comment by freakboy3742** [comment]
The tutorial has now been updated to include details for Android builds.
